dockerfiles/Dockerfile_full

```diff
@@ -2,22 +2,24 @@ FROM heroku/heroku:18
 
 ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update
-RUN echo 'f6d5a254de2734ad9e7b9b02777d47a9a7454059e2d5690cf25c5ea57280f060  ./nodejs.deb' > node-file-lock.sha \
-  && curl -s -o nodejs.deb https://deb.nodesource.com/node_10.x/pool/main/n/nodejs/nodejs_10.16.2-1nodesource1_amd64.deb \
+RUN echo '2d316e55994086e41761b0c657e0027e9d16d7160d3f8854cc9dc7615b99a526  ./nodejs.tar.gz' > node-file-lock.sha \
+  && curl -s -o nodejs.tar.gz https://nodejs.org/dist/v12.18.2/node-v12.18.2-linux-x64.tar.gz \
   && shasum --check node-file-lock.sha
+RUN mkdir /usr/local/lib/nodejs \
+  && tar xf nodejs.tar.gz -C /usr/local/lib/nodejs/ --strip-components 1 \
+  && rm nodejs.tar.gz node-file-lock.sha
```

Comment:
The process for both slim and full is:
1. Download the tar.gz file form nodejs.org
1. Check the hash of the tar.gz to make sure we are getting the correct file. See README for instructions on `checksum`
1. Create `/usr/local/lib/nodejs` and unszip node source into that directory.
1. Add Node bin to PATH.